src/client.d.ts

```diff
@@ -193,15 +195,17 @@ declare namespace deepstream {
 
 	class RecordHandler {
 		getAnonymousRecord<T>(): AnonymousRecord<T>
-		getList(name: string, options?: any): List
+		getList<T>(name: string, options?: any): List<T>
 		getRecord<T>(name: string): Record<T>
+		setRecord<T, D>(name: string, data: D, callback?: EventCallbackFn<T>): void
```

Comment:
@roblav96 UP on that and @AlexBHarley  can we expect a soonish merge ?

best regards